app/web/components/OldMap.tsx

```diff
@@ -0,0 +1,130 @@
+import "maplibre-gl/dist/maplibre-gl.css";
+
+import { styled, Typography } from "@mui/material";
+import { NO_MAP_SUPPORT } from "components/constants";
+import {
+  LngLat,
+  Map as MaplibreMap,
+  NavigationControl,
+  RequestParameters,
+} from "maplibre-gl";
+import { useEffect, useRef, useState } from "react";
+
+const URL = process.env.NEXT_PUBLIC_API_BASE_URL;
+
+const StyledWrapper = styled("div")<{ grow?: boolean }>(({ grow }) => ({
+  position: "relative",
+  height: grow ? "100%" : "200px",
```

Comment:
maybe constants for the 200px and 400px values?

Reply:
and reuse them in different files

Reply:
This is the old map so I don't change anything here.

Reply:
I do have size constants for the new map though!